examples/3_simple_storage/simple_storage.go

```diff
@@ -0,0 +1,231 @@
+/*
+WHAT IS IN THIS EXAMPLE?
+
+Keybase has added an encrypted key-value store intended to support
+security-conscious bot development with persistent state. It is a
+place to store small bits of data that are
+  (1) encrypted for a team or user (via the user's implicit self-team: e.g.
+alice,alice),
+  (2) persistent across logins
+  (3) fast and durable.
+
+It supports putting, getting, listing, and deleting. There is also
+concurrency support, but check out 5_secret_storage for that. A team has many
+namespaces, a namespace has many entryKeys, and an entryKey has one current
```

Comment:
is there a doc on the website to link to? Would likely be easier to have the description fully fleshed out in a single place rather than spread across the libraries/examples.

Reply:
agree. here's a ticket for it https://keybase.atlassian.net/browse/PICNIC-640